src/3D/Camera.cpp

```diff
@@ -131,7 +146,7 @@ void Camera::ProcessSDLEvent(const SDL_Event& e)
 {
 	if (e.type == SDL_KEYDOWN || e.type == SDL_KEYUP)
 		handleKeyboardInput(e);
-	else if (e.type == SDL_MOUSEMOTION || e.type == SDL_MOUSEBUTTONDOWN || e.type == SDL_MOUSEBUTTONUP)
+	else if (e.type == SDL_MOUSEMOTION || e.type == SDL_MOUSEBUTTONDOWN || e.type == SDL_MOUSEBUTTONUP || SDL_MOUSEWHEEL)
```

Comment:
`e.type == SDL_MOUSEWHEEL`

Reply:
Might even be the time to move to a switch statement:

```c++
switch (e.type)
{
    case SDL_KEYDOWN:
    case SDL_KEYUP:
        handleKeyboardInput(e);
        break;
    case SDL_MOUSEMOTION :
    case SDL_MOUSEBUTTONDOWN :
    case SDL_MOUSEWHEEL:
        handleMouseInput(e);
        break;
        break;
}
```

Reply:
yes good point I'll change this.